configs/satellite_config.yaml

```diff
@@ -140,4 +140,28 @@ pipes:
       min_flush_events: ${SATELLITE_TRACINGPIPE_SENDER_MIN_FLUSH_EVENTS:1}
       client_name: grpc-client
       forwarders:
-        - plugin_name: nativetracing-grpc-forwarder
\ No newline at end of file
+        - plugin_name: nativetracing-grpc-forwarder
+  - common_config:
+      pipe_name: profilepipe
+    gatherer:
+      server_name: "grpc-server"
+      receiver:
+        plugin_name: "grpc-nativeprofile-receiver"
+      queue:
+        plugin_name: "memory-queue"
```

Comment:
This doesn't look like `none queue mode`. Could you be clear your meaning?

Reply:
in the profiling protocol, we also need to use a queue to transform thread dumps.

Reply:
> Using none queue mode to support the SkyWalking native profile channel

What does this comment mean?